<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrator</class>
 <widget class="QMainWindow" name="Administrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#administratorWidget,#workerWidget,#consumerWidget,#orderWidget,#carWidget{
border-image:url(:/HX/images/管理员.jpg)
}

QTabWidget::tab-bar {
    left: 5px;
    alignment: center;
}

QTabWidget::pane {
    border: 1px solid #9B9B9B;
    position: absolute;
    top: -1px;
}

QTabBar::tab {
    background: QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #E1E1E1,stop:0.4 #DDDDDD,stop:0.5 #D8D8D8,stop:1 #D3D3D3);
    border-left: 1px solid #C4C4C3;
    border-right: 1px solid #C4C4C3;
    border-top: 1px solid #C4C4C3;
    border-bottom: 1px solid #9B9B9B;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 100px;
    padding: 5px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #fafafa,stop:0.4 #f4f4f4,stop:0.5 #e7e7e7,stop:1 #fafafa);
}

QTabBar::tab:selected {
    border-left-color: #9B9B9B;
    border-right-color: #9B9B9B;
    border-top-color: #9B9B9B;
    border-bottom-color: #fafafa;
    margin-left: -3px;
    margin-right: -3px;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QTabBar::tab:first:selected {
    margin-left: 0;
}

QTabBar::tab:last:selected {
    margin-right: 0;
}

QTabBar::tab:only-one {
    margin: 0;
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="administratorWidget">
       <attribute name="title">
        <string>管理员信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="adminView">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="workerWidget">
       <attribute name="title">
        <string>员工信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="workerView"/>
        </item>
        <item>
         <widget class="QChartView" name="WorkerView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="consumerWidget">
       <attribute name="title">
        <string>客户信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="consumerView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderWidget">
       <attribute name="title">
        <string>订单信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableView" name="orderView"/>
        </item>
        <item>
         <widget class="QChartView" name="OrderView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="carWidget">
       <attribute name="title">
        <string>车辆信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTableView" name="carView"/>
        </item>
        <item>
         <widget class="QChartView" name="CarView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actSave"/>
   <addaction name="actCancel"/>
   <addaction name="separator"/>
   <addaction name="actExport"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
   <addaction name="separator"/>
   <addaction name="actSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/316.bmp</normaloff>:/HX/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/107.bmp</normaloff>:/HX/images/107.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/save1.bmp</normaloff>:/HX/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/delete1.bmp</normaloff>:/HX/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/exit.bmp</normaloff>:/HX/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actExport">
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/save1.bmp</normaloff>:/HX/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出数据表</string>
   </property>
  </action>
  <action name="actSelect">
   <property name="icon">
    <iconset resource="HX.qrc">
     <normaloff>:/HX/images/324.bmp</normaloff>:/HX/images/324.bmp</iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="toolTip">
    <string>查询</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HX.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>Administrator</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
